lib/upload.js

```diff
@@ -276,13 +284,23 @@ class Upload {
 
     cb = cb || function () {};
     if (shouldTerminate && this.url != null) {
-      Upload.terminate(this.url, this.options, (err, xhr) => {
-        if (err) {
-          return cb(err, xhr);
-        }
+      const terminate = () => {
+        Upload.terminate(this.url, this.options, (err, xhr) => {
```

Comment:
It would be nice if the `Upload.terminate` function would be able to retry on its own, so you can benefit from the retries without having an instance of the `tus.Upload` class. Could you look into if that's easily possible?